extensions/usage-metrics-dashboard/get_usage.R

```diff
@@ -1,5 +1,4 @@
 library(connectapi)
-library(pins)
```

Comment:
Was this leftover cruft? Or did we change something here that I'm not realizing removes the need?

Reply:
Yes, it was leftover cruft that I missed when deleting the pins usage in an old PR.